autopush/endpoint.py

```diff
@@ -395,6 +395,9 @@ class EndpointHandler(AutoendpointHandler):
                             "encryption-key", "content-type",
                             "authorization"]
     cors_response_headers = ["location", "www-authenticate"]
+    # Remove trailing padding characters from complex header items like
+    # Crypto-Key and Encryption
+    strip_pad = re.compile("=+(?=[,;]|$)")
```

Comment:
I think we'll want to account for the quote (`[,;"]`), too, in case they pass a quoted string.